service/src/main/resources/config.template.properties

```diff
@@ -11,13 +11,13 @@
   logging.level.com.ericsson.eiffel.remrem.producer: ERROR
 
 # Jasypt configurations
-  jasypt.encryptor.password:<Any value which was used at the time of encrypting the original password will be used at runtime to decrypt the same>
+ jasypt.encryptor.jasyptKeyFilePath: <The path of jasypt.key file containing the key value which was used at the time of encrypting the original password will be used at runtime to decrypt the same>
 
 #Ldap authentication configurations
 
   activedirectory.generate.enabled:false 
   activedirectory.ldapUrl : 
-  activedirectory.managerPassword : ENC(<encrypted password>) or password
+  activedirectory.managerPassword : {ENC(<encrypted password>)} or password
```

Comment:
add a comment, if ENC is used then it will need the jasypt.key path otherwise it will take the unencrypted passwords.

Reply:
done